src/sidebar/components/ModerationBanner.tsx

```diff
@@ -68,6 +68,8 @@ function ModerationBanner({
     <div
       className={classnames(
         'flex gap-x-3 bg-grey-1 text-color-text font-semibold',
+        // Match the card's border radius and hide overflow
+        'rounded-t-lg overflow-hidden',
```

Comment:
We have used `rounded-[inherit]` in other places, to avoid coupling container styles with their children. However, in this case there are a few intermediate DOM nodes that make to not work.